tools.yml

```diff
@@ -7,7 +7,7 @@ refactoring_miner: ~/RefactoringMiner-1.0/bin/RefactoringMiner
 # https://github.com/boyter/scc
 scc: ~/scc/scc
 # universal-ctags
-utags: /usr/local/Cellar/universal-ctags/HEAD-62f0144/bin/ctags
```

Comment:
have to be careful with these. GitHub Actions use them. This could be the source of some checks failing.

Reply:
The path that is merged on main is the same as this. What should it be changed to?

Reply:
Oops... I guess I confused myself here. Please go ahead and revert to what main has. I actually think I may have broken the Actions doing this, sorry! See if this makes both checks pass!